### PR TITLE
Add locate projects

### PR DESCRIPTION
**Pull request by BillWagner**
Contributes to #99 

This may be easier to review commit-by-commit. 

In this PR, I created a new global solution that builds all tools in this repository. That makes it easier to find duplicate code.
Next, I added the LocateProjects project and its dependencies into this repo. No functional changes, just a straight move.
Finally, I removed duplicated code where we'd copied some utility code while creating the Quest2GitHub project.
Then, update the version on all affected projects.

### CODE REVIEW

File: DotNet.DocsTools/GitHubCommunications/GitHubClient.cs [hunk 1]
```diff
@@ -1,47 +1,32 @@
...
-using Polly.Retry;
-using System;
+﻿using System.Net.Http.Headers;
+using System.Net.Mime;
+using System.Text.Json;
 
 namespace DotnetDocsTools.GitHubCommunications;
 
-public sealed class GitHubClient : IGitHubClient, IDisposable
+internal class GitHubClient : IGitHubClient
```
Comment (IEvangelist) [inline DotNet.DocsTools/GitHubCommunications/GitHubClient.cs:7]: This entire file should be reverted. The changes that these overwrite are undesirable, and remove the transient fault handling that we paired on adding in #92.

File: DotNet.DocsTools/GitHubCommunications/MarkdownToHTMLRequest.cs [hunk 2]
```diff
@@ -1,19 +1,13 @@
-﻿namespace DotnetDocsTools.GitHubCommunications;
+﻿using System.Text.Json;
+
+namespace DotnetDocsTools.GitHubCommunications;
```
Comment (IEvangelist) [inline DotNet.DocsTools/GitHubCommunications/MarkdownToHTMLRequest.cs:3]: I'd probably revert this entire file too, as it is more complete than the proposed changes.

**Review by IEvangelist — CHANGES_REQUESTED** [review]
This looks good thus far, but there are two files that should be reverted. Otherwise, let's :shipit:

File: DotNet.DocsTools/GitHubCommunications/GitHubClient.cs (hunk 1 above)
Comment (BillWagner) [reply DotNet.DocsTools/GitHubCommunications/GitHubClient.cs]: I made the version with Polly the shared version.

File: DotNet.DocsTools/GitHubCommunications/MarkdownToHTMLRequest.cs (hunk 2 above)
Comment (BillWagner) [reply DotNet.DocsTools/GitHubCommunications/MarkdownToHTMLRequest.cs]: It may come back, but reverted and commented out the one API where it is called.

**Comment by BillWagner** [comment]
Thanks @IEvangelist 

This is ready for a second look

**Review by IEvangelist — APPROVED** [review]
Looks good to me, :shipit: when you're ready.

**Pull request merged**